eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -159,9 +159,9 @@ stages:
                         displayName: Set Documentation File Name
                       - template: /eng/pipelines/templates/steps/stage-artifacts.yml
```

Comment:
Why do we need the stage-artifacts.yml any more? I'd like to eliminate it now if possible.